src/main/rx/spektrum.c

```diff
@@ -70,6 +70,11 @@ static bool spekHiRes = false;
 static bool srxlEnabled = false;
 static int32_t resolution;
 
+#ifdef USE_SPEKTRUM_RSSI_CH
+// This is for spektrum receivers that use one of the channels for RSSI.
+#undef USE_SPEKTRUM_FAKE_RSSI
```

Comment:
Wrong, You are turning this feature off  globally for every target when doing it here. If you do not want it, you should should undef it in your target specific target.h file.